### ISSUE TITLE
free user

### ISSUE THREAD

**Issue by codsec** [ONLYOFFICE/CommunityServer#435]
how many connection/user can i have for free?

**Comment by Carazyda** [comment]
Hello @codsec. You can add 10,000 users to CommunityServer. Regarding connections to DocumentServer, the free version has a limit of 20 connections.

**Comment by codsec** [comment]
@Carazyda is the connection limited for each open doc? or is it per user?

**Comment by Carazyda** [comment]
Yes, for each open document.

**Comment by codsec** [comment]
> Yes, for each open document.

ok got it, one last thing, if i buy the home version, i will have 20 free connections and more 10? or nah?
![image](https://user-images.githubusercontent.com/44205014/203635565-b244340f-63a3-4b11-94b9-6871a38055da.png)

**Comment by Carazyda** [comment]
No, 10 users can open documents at the same time, there is no limit to the number of documents for each user. For more information on licenses, please contact sales@onlyoffice.com.

**Comment by codsec** [comment]
oh thats good, but Is there a way or possibility for me to have 20 users with this home plan?

**Comment by Carazyda** [comment]
I think you can come to an agreement. Please contact our sales department sales@onlyoffice.com.

**Comment by dwpoint** [comment]
> I think you can come to an agreement. Please contact our sales department [sales@onlyoffice.com](mailto:sales@onlyoffice.com).

@Carazyda Are you talking about limits in the cloud or in general?
Is there a limit on the number of coauthoring users per document for a local server on Windows 2016 x86?

**Comment by Carazyda** [comment]
@dwpoint here we were talking about the free version of Onlyoffice Workspace, and the minimum license for the document server.

In the workspace, you can only open 20 tabs with document editors.

**Comment by dwpoint** [comment]
@Carazyda  Okay These are tabs. But if the document is 1. It turns out that 20 users will be able to edit it at the same time? I'm talking about co-editing.

**Comment by dwpoint** [comment]
@Carazyda The thing is, I kind of downloaded the community version for my local Windows server, but at the same time I have a rebranding available, I can adjust the size of the repository and in general, it seems I have all the details. Is it possible to see somewhere on the control panel which version I have installed at the moment? Paid or free.

**Comment by codsec** [comment]
Let me see if i understanding... if i selfhosted the source still has limitation? or is unlimited when im self hosting?

**Comment by dwpoint** [comment]
> Let me see if i understanding... if i selfhosted the source still has limitation? or is unlimited when im self hosting?

A very good and valid question. Let's wait for the answer. I'm curious too.

**Comment by codsec** [comment]
> 

yeah, for me it wouldn't make sense to have a limitation since the host would be myself, and the service is really great but i avoid using because of that limitation, but i never tested on self hosted...so if there is not limitation on self hosted that would be a great thing

**Comment by Carazyda** [comment]
> It turns out that 20 users will be able to edit it at the same time?
Yes, you are right. 21 user will open the document viewer. And only twenty users will be able to edit this document at the same time. Any other document at this moment will be opened for viewing.

> Is it possible to see somewhere on the control panel which version I have installed at the moment? Paid or free.

You can see at the payments page in ControlPanel - /controlpanel/activate

> Let me see if i understanding... if i selfhosted the source still has limitation? or is unlimited when im self hosting?
Limitations in the Document Server that I described above.
